django/core/cache/backends/memcached.py

```diff
@@ -147,7 +147,9 @@ def set_many(self, data, timeout=DEFAULT_TIMEOUT, version=None):
         for key, value in data.items():
             key = self.make_key(key, version=version)
             safe_data[key] = value
-        self._cache.set_multi(safe_data, self.get_backend_timeout(timeout))
+        failing_keys = self._cache.set_multi(safe_data,
+                                             self.get_backend_timeout(timeout))
+        return failing_keys if failing_keys else None
```

Comment:
Is `else None` to convert an empty list to `None`? If not sure whether or not it's a good design, but it would be helpful to document the design decision it.